### DISCUSSION TITLE
How can I change openapi.json?

### DISCUSSION THREAD

**Discussion by vadim-su** [fastapi/fastapi#7794] [Questions] [answered]
I try to add an optional field to an output schema and I get openapi.json with this field without a nullable flag.
```
class OutSchema(BaseModel):
    ....
    removed_at: Optional[datetime.datetime]
```
```
"removed_at": {
            "title": "Removed At",
            "type": "string",
            "format": "date-time"
}
```
How can I add the flag to openapi.json?
It's important for me because I use openapi.json to generate C# clients.

**Comment by phy25** [comment, marked as answer]
See #833.

**Comment by vadim-su** [comment]
Thanks!

**Comment by tiangolo** [comment]
Thanks for the help @phy25 ! :cake: :bowing_man: 

Thanks @deterok for reporting back and closing the issue :+1:
